.github/workflows/release.yml

```
@@ -18,8 +18,12 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
     steps:
-      - uses: nearform/optic-release-automation@main
+      - uses: nearform-actions/optic-release-automation@main
```

@bredikhin this was probably an old rename, which maybe worked until the repo was transferred. 

```suggestion
      - uses: nearform-actions/optic-release-automation-action@main
```